build/upgrade_pdfjs.sh

```diff
@@ -50,16 +51,13 @@ echo "Enabling e-signature rendering without validation..."
 echo "-----------------------------------------------------------------------------------"
 sed -e 's@;r.setFlags(o.AnnotationFlag.HIDDEN)@@' -i '' ${DOC_STATIC_ASSETS_PATH}/pdf.worker.min.js
 
-# Fix for Courier font PDF not rendering applied to src/third-party/doc/2.76.0/pdf.worker*.js which is PDFJS v2.2.228
-# Link to issue: https://github.com/mozilla/pdf.js/issues/13771
```

Comment:
Did we verify this works with the target version of pdfjs still?

Reply:
Seems like it still works! Was able to preview a PDF with Courier font in IE11 (v2.2.228) with no issues